specification/columns/tags.md

```diff
@@ -6,30 +6,40 @@ A tag becomes [*finalized*](#glossary:finalized-tag) when a single value is sele
 
 The Tags column adheres to the following requirements:
 
-* The Tags column MUST be present in a [*FOCUS dataset*](#glossary:FOCUS-dataset) when the provider supports setting user or provider-defined tags.
-* The Tags column MUST contain user-defined and provider-defined tags.
-* The Tags column MUST only contain finalized tags.
-* The Tags column MUST be in [KeyValueFormat](#key-valueformat).
+* Tags MUST be present in a [*FOCUS dataset*](#glossary:FOCUS-dataset) when the provider supports setting user or provider-defined tags.
+* Tags MUST contain all user-defined and provider-defined tags.
+* Tags MUST only contain finalized tags.
+* Tags MUST be in [KeyValueFormat](#key-valueformat).
 * A Tag key with a non-null value for a given resource SHOULD be included in the tags column.
 * A Tag key with a null value for a given resource MAY be included in the tags column depending on the provider's tag finalization process.
 * A Tag key that does *not* support a corresponding value, MUST have a corresponding true (boolean) value set.
+* Providers MUST NOT alter Tag values unless applying true (boolean) to valueless tags
 * If Tag finalization is supported, providers MUST publish tag finalization methods and semantics within their respective documentation.
-* Providers MUST NOT alter user-defined Tag keys or values.
+* Providers MUST NOT allow reserved tag key prefixes to be used as prefixes for any user-defined tag keys within a prefixless, user-defined tag scheme.  
+* Providers SHOULD publish all provider-specified tag key prefixes within their respective documentation.
```

Comment:
I forget... Don't we require defining prefixes in metadata? Can we make this a MUST and switch docs to metadata? 
```suggestion
* Providers MUST publish all provider-specified tag key prefixes within FOCUS metadata.
```

Reply:
We intentionally left this open-ended to not dictate firm guidance beyond the FOCUS spec itself.  If a provider really doesn't (want to) publish this documentation, then it's on them.

Reply:
Resolving in preparation for the member's vote. Reopen if needed though.